go.mod

```diff
@@ -7,12 +7,10 @@ require (
 	github.com/Azure/go-autorest v10.8.1+incompatible // indirect
 	github.com/Shopify/logrus-bugsnag v0.0.0-20171204204709-577dee27f20d
 	github.com/aws/aws-sdk-go v1.34.9
-	github.com/bitly/go-simplejson v0.5.0 // indirect
 	github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869 // indirect
 	github.com/bshuster-repo/logrus-logstash-hook v0.4.1
-	github.com/bugsnag/bugsnag-go v0.0.0-20141110184014-b1d153021fcd
-	github.com/bugsnag/osext v0.0.0-20130617224835-0dd3f918b21b // indirect
-	github.com/bugsnag/panicwrap v0.0.0-20151223152923-e2c28503fcd0 // indirect
+	github.com/bugsnag/bugsnag-go v2.1.0+incompatible // indirect
```

Comment:
I'm a bit confused by this (`// indirect` dependency) given we depend on `github.com/bugsnag/bugsnag-go/v2` of the same tag, but maybe I'm missing something here about `Go` modules. Did you run `go mod tidy`?

Reply:
Yes, I did:
```
distribution ❯ go mod tidy
go: finding module for package github.com/bugsnag/bugsnag-go
go: finding module for package github.com/bugsnag/bugsnag-go/errors
go: found github.com/bugsnag/bugsnag-go in github.com/bugsnag/bugsnag-go v2.1.0+incompatible
go: found github.com/bugsnag/bugsnag-go/errors in github.com/bugsnag/bugsnag-go v2.1.0+incompatible

```

Reply:
I *think* if you remove `github.com/bugsnag/bugsnag-go/v2 v2.1.0` from go mode and update github.com/bugsnag/bugsnag-go v2.1.0+incompatible`  -> `github.com/bugsnag/bugsnag-go v2.1.0`, then after `go get` if you run `go mod tidy` things would work as expected and be a bit cleaner 🤔

Reply:
I tried but it doesn't help.

I believe it adds `github.com/bugsnag/bugsnag-go v2.1.1+incompatible // indirect`  because of dependencies in `github.com/Shopify/logrus-bugsnag` package, which has these:

https://github.com/Shopify/logrus-bugsnag/blob/577dee27f20dd8f1a529f82210094af593be12bd/bugsnag.go#L7-L8

So, without updating `logrus-bugsnag` to Bugsnag `v2` we can't bump Bugsnag itself.